### PR TITLE
Update kafka-trigger-tutorial.md

### PR DESCRIPTION
**Pull request by viveksinghggits**
There wasnt a step if in case the Kafka is running somewhere else.

### CODE REVIEW

File: docs/content/usage/kafka-trigger-tutorial.md [hunk 1]
```diff
@@ -126,6 +126,8 @@ We have both the functions ready but the connection between them is the missing
 $ fission mqt create --name kafkatest --function consumerfunc --mqtype kafka --topic input --resptopic output --errortopic error
 ```
 
+If your Kafka in running somewhere else (in some other machine), you will have to provide custom configuration for Kafka broker host while installing fission. You can do that by creating a config file, set the value of `kafka.brokers` to you broker URL and provide this config file while installing fission through helm using -f flag. You can refer this [link](https://github.com/fission/fission/blob/master/charts/fission-all/values.yaml) to find out more about this config parameter.
```
Comment (bhavin192) [inline docs/content/usage/kafka-trigger-tutorial.md]: Even if the Kafka instance is running in same cluster and or same namespace, one still have to specify correct FQDN of Kafka service in the `values.yaml` while installing the helm chart.

Comment (viveksinghggits) [reply docs/content/usage/kafka-trigger-tutorial.md]: So we can simply say that if your Kafka broker is not running at `broker.kafka:9092`, you will have to specify this configuration while installing fission.

File: docs/content/usage/kafka-trigger-tutorial.md [hunk 2]
```diff
@@ -126,6 +126,8 @@ We have both the functions ready but the connection between them is the missing
 $ fission mqt create --name kafkatest --function consumerfunc --mqtype kafka --topic input --resptopic output --errortopic error
 ```
 
+If your Kafka broker is running somewhere else (not at `broker.kafka:9092`), you will have to provide custom configuration for Kafka broker host while installing fission. You can do that by creating a config file, set the value of `kafka.brokers` to you broker URL and provide this config file while installing fission through helm using -f flag. You can refer this [link](https://github.com/fission/fission/blob/master/charts/fission-all/values.yaml) to find out more about this config parameter.
```
Comment (life1347) [inline docs/content/usage/kafka-trigger-tutorial.md]: Minor fix: ...to your broker URL and...

Comment (viveksinghggits) [reply docs/content/usage/kafka-trigger-tutorial.md]: done

**Comment by life1347** [comment]
LGTM

**Pull request merged**